app/assets/components/profile-accounts/profile-accounts.css.scss

```diff
@@ -0,0 +1,42 @@
+@import 'common';
+@import 'imports';
+
+:host {
+  display: inline-block;
+  margin: 18px;
+  width: 350px;
+  height: auto;
+
+  header {
+    font-size: 18px;
+    padding: 8px 14px;
+    background-color: $accent;
+    color: $text-icons;
+  }
+
+  .danger {
+    color: $danger;
+  }
+
+  .accounts-list {
+    padding: 18px;
+
+    div {
+      margin: 4px 0px;
+    }
+
+    .actions {
+      margin-left: 4px;
+    }
+
+    a {
+      cursor: pointer;
+      color: $secondary-text;
+      text-decoration: none;
+      &:hover {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3